### ISSUE TITLE
display CD number like Navidrome

### ISSUE THREAD

**Issue by Yumega** [owntone/owntone-server#1629]
The navidrome can display the CD number of an album, like this 
![image](https://github.com/owntone/owntone-server/assets/61570667/3a2b02e7-c535-4a2c-912e-22ce6b9ecd77)
![image](https://github.com/owntone/owntone-server/assets/61570667/c2fee5ca-6cad-49b2-a810-1e2a9b1c604f)

but daapd don't display the CD numer
![image](https://github.com/owntone/owntone-server/assets/61570667/000711da-0870-4708-a889-0e786a2faa65)

**Comment by hacketiwack** [comment]
@ejurgensen and @Yumega Could something like this be acceptable (see below) (alignment is incorrect, though)?

Question is: what information should we keep in the list?
In the example below, I removed the track artist and the album title. Moreover, I added the track number in the media and the track duration. I, somehow, replicated what is visible in Apple Music.

![Media Number](https://github.com/owntone/owntone-server/assets/13329768/f9b73c47-6879-446b-be5b-0b097cd48916)

**Comment by hacketiwack** [comment]
@ejurgensen, I'm also wondering if we could retrieve the medium subtitle when it is present in the metadata of the track.

Typically, MusicBrainz Picard is adding this information in the field `DISCSUBTITLE`:  Example [here](https://musicbrainz.org/release/c1144167-216f-4b43-a2a6-7f68416ed481).

**Comment by ejurgensen** [comment]
Yes, I think it would be possible. The DISCSUBTITLE in the example is "Deux Enfants au soleil", right?

With regard to the interface, it's of course a matter of taste, but I would vote for keeping the current clean look. I don't think there is much value in showing the track number nor the duration in the main view.

CD number and CD title/subtitle seem a bit antiquated to me, but if you would like to have them, then I think the subtitles you illustrated above look ok.

**Comment by hacketiwack** [comment]
@ejurgensen, I fully agree with you. Not too much information must be visible on this interface to avoir cluttering it.
I would appreciate to have the media subtitle (as you've written, "Deux Enfants au soleil", is the subtitle), because it brings more information. But the disc number not that much to be honest. As you say, it is quite outdated nowadays and is anyway visible in the details of the track.

**Comment by hacketiwack** [comment]
For the moment, because we don't have the DISCSUBTITLE yet, I will display the medium number only if there are more than one medium in the album.
